packages/playwright-core/src/server/chromium/chromium.ts

```diff
@@ -275,7 +275,7 @@ export class Chromium extends BrowserType {
 
   override defaultArgs(options: types.LaunchOptions, isPersistent: boolean, userDataDir: string): string[] {
     const chromeArguments = this._innerDefaultArgs(options);
-    chromeArguments.push(`--user-data-dir=${userDataDir}`);
+    chromeArguments.push(`--user-data-dir=${path.resolve(userDataDir)}`);
```

Comment:
This is going to be relative to the driver's location in language ports, isn't it? I believe we want it to be relative to the client executable, which means computing it on the client.

Reply:
Its relative to the CWD of where the driver was started. In all of the ports, we don't specify the CWD, so it inherits the CWD of the parent process:

- [Python](https://github.com/microsoft/playwright-python/blob/d3aacbb7dccbd39b37965013070d95a043ed0f3e/playwright/_impl/_transport.py#L120)
- [.NET](https://github.com/microsoft/playwright-dotnet/blob/390f50e53259065c86db8111403590a26512e735/src/Playwright/Transport/StdIOTransport.cs#L117C29-L117C45)
- [Java](https://github.com/microsoft/playwright-java/blob/995cf902fb09fd54ea512eab06101cc8d62d5009/driver/src/main/java/com/microsoft/playwright/impl/driver/Driver.java#L74)

Sounds reasonable to me?